### DISCUSSION TITLE
Does it support query.Include(u => u.Roles)?

### DISCUSSION THREAD

**Discussion by fpdave100** [ramantsitou/MockQueryable#71] [Q&A] [answered]
Its only a single query or dbset object, so I would be surprised if this was going to be supported.
I am having some issues, but dont know if they are related to this mocking

**Comment by ramantsitou** [comment, marked as answer]
Hello @fpdave100. Unfortunately query.Include( *** ) is not supported at the moment. You can see the discussion in #45 . Feel free to create a pull request if you have some ideas how to make it supported
